cli/cmd/build/config/config.go

```diff
@@ -15,13 +16,49 @@ type Artifact struct {
 	URL  string `yaml:"url"`
 }
 
+type Extension struct {
+	Name    string            `yaml:"name"`
+	Version string            `yaml:"version"`
+	Specs   map[string]string `yaml:"specs"`
```

Comment:
we load in a generic struct

```suggestion
	Specs   any `yaml:"specs"`
```